src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -612,7 +612,7 @@ def BuildDpa(dpa_name, protection_points_method=None):
     monitor_type = 'esc'
   except KeyError:
```

Comment:
This error won't be raised if the name of the portal dpa provided in the config file (field dpaId) happens to exist in the default ESC NTIA file.

=> to avoid ending up with the wrong portal DPA points, it may be safer not to call GetCoastalDpaZones when building a portal DPA (e.g, by adding an optional parameter "isPortalDpa" to BuildDpa)

Reply:
ok to leave the code unchanged. I agree this situation is not supposed to happen & can be easily detected. Thanks!